package.json

```diff
@@ -35,8 +35,12 @@
     "@redhat-cloud-services/frontend-components-notifications": "^3.2.2",
     "@redhat-cloud-services/frontend-components-utilities": "^3.2.4",
     "classnames": "^2.3.1",
+    "dayjs": "^1.10.8",
+    "file-saver": "1.3.x",
     "formik": "^2.2.9",
     "history": "4.x",
+    "i18next": "^19.8.3",
```

Comment:
Where is this being used? Doesn't `react-i18next` give us enough utilities around translations?

Reply:
@rohitkrai03 this is for utilities. `i18next.t()`